pkg/container/client.go

```diff
@@ -652,6 +654,11 @@ func (client dockerClient) tcContainerCommand(ctx context.Context, target *Conta
 	return nil
 }
 
+
+func GetUnexportedField(field reflect.Value) string {
```

Comment:
Please, do not use reflection to expose the internal field.
Consider doing this explicitly. 
- Add a new `host-socket` parameter to the stress command (`NewStressCLICommand`) 
- Default to `/var/run/docker.sock`
- Pass it to the `stressContainerCommand` function